### PR TITLE
FAQ: Where are downloaded files stored?

### PR DESCRIPTION
**Pull request by MinekPo1**
Not entirely sure if this is the right amount of detail, but I didn't want to infodump here for no reason.

Took over from @DMenezes on matrix.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -226,6 +226,19 @@ <h2>Experience the difference</h2>
...
             </p>
           </details>
+
+          <details>
+            <summary>Where are downloaded files stored?</summary>
+            <p>
+              Downloaded files are stored in external private storage
+              (<code>Android/data/com.github.libretube</code>).<br/>
+              Note that since Android 11, the <code>READ_EXTERNAL_STORAGE</code>
+              permission does not give access to external private storage,
```
Comment (Bnyro) [inline index.html]:
```suggestion
              permission does not grant third party apps access to external private storage,
```

File: index.html [hunk 2]
```diff
@@ -226,6 +226,19 @@ <h2>Experience the difference</h2>
...
+              requiring workarounds.<br/>
+              The (usually hidden, not to be confused with Files by Google)
+              Files app will always have access to this directory.
```
Comment (Bnyro) [inline index.html]:
```suggestion
              Android-builtin Files app will always have access to this directory.
```

**Comment by Bnyro** [comment]
I'd also add something like
`In the downloaded files, audio and video are separated. Hence it is recommended to exclusively use LibreTube for playing downloaded media.`

**Comment by MinekPo1** [comment]
sorry for taking three days, but not in the best of places mentally as of late, haha.

**Review by Bnyro — APPROVED** [review]
Thanks!

**Comment by Bnyro** [comment]
> sorry for taking three days, but not in the best of places mentally as of late, haha.
>
No worries :)

**Pull request merged**